### DISCUSSION TITLE
Upgrading NATS from v2.8.0 to v2.9.17

### DISCUSSION THREAD

**Discussion by beatrausch** [nats-io/nats-server#4201] [Q&A]
Hi,

we want to update our NATS cluster that runs on K8 and has been installed with Helm to version v2.9.17. 
Do we have to take care about breaking changes, that requires migration via an explicit version. 

Thx, Regards

**Comment by derekcollison** [comment]
Depends on the complexity of your system. Best to consult with someone from Synadia prior.
